.github/workflows/generate-summary.yaml

```diff
@@ -406,6 +406,20 @@ jobs:
             -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
             https://api.github.com/rate_limit
 
+      - name: Create Issue Gist
+        run: |
+          python3 -m pip install pygithub requests pyopenssl --upgrade
+          FILE_SIZE=$(stat -c%s "issue.md")
+          echo "file_size : $FILE_SIZE"
+          ONE_MB_IN_BYTES=1048576
+          head -c $ONE_MB_IN_BYTES "issue.md" > gist_trimmed_issue.md
+          if [ "$FILE_SIZE" -gt "$ONE_MB_IN_BYTES" ]; then
+              echo "The file was greater than 1 MB and has been trimmed to fit the gist limit. Please check logs for the untrimmed issue."
```

Comment:
Can we trim the gist here so it can still be uploaded and include this message in the gist?

Reply:
This is a test gist to see if the gist are created normally. After stabilization, I am planning on adding a proper trim message to the uploaded contents